ls1intum/Artemis
# [Bugfix] Fix issue when submitting modeling example assessment with feedback credits of 0

### Checklist

### Motivation and Context


Fixes #2494. Issue was again that `0` evaluates to false

### Description

- Added undefined check 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new modeling exercise or use old one and edit it
4. Create example submission
5. Create example assessment with at least one feedback having credits of 0
6. Click on save example assessment and check that everything works as expected
7. Try also different feedback credits  combinations

### Test Coverage



Nothing changed

### Screenshots


<img width="1659" alt="image" src="https://user-images.githubusercontent.com/41108487/101697644-78aca900-3a78-11eb-8f62-ed1342b51c6c.png">
<img width="1659" alt="image" src="https://user-images.githubusercontent.com/41108487/101697866-d640f580-3a78-11eb-9128-c4c7d7298380.png">
